raft/src/main/java/org/apache/kafka/raft/QuorumState.java

```diff
@@ -548,6 +550,10 @@ public VotedState votedStateOrThrow() {
             .orElseThrow(() -> new IllegalStateException("Expected to be Voted, but current state is " + state));
     }
 
+    public Optional<Node> idToNode(int id, String listener) {
+        return latestVoterSet.get().idToNode(id, listener);
+    }
```

Comment:
See my other comment but I think we can remove this.

Reply:
removed